1-js/99-js-misc/06-unicode/article.md

```diff
@@ -1,172 +1,167 @@
 
-# Unicode, String internals
+# Unicode, detalhes internos de Strings
 
-```warn header="Advanced knowledge"
-The section goes deeper into string internals. This knowledge will be useful for you if you plan to deal with emoji, rare mathematical or hieroglyphic characters, or other rare symbols.
+```warn header="Conhecimento avançado"
+Esta seção aprofunda nos detalhes internos de strings. Este conhecimento será útil se você planeja lidar com emojis, caracteres matemáticos raros, hieróglifos ou outros símbolos raros.
 ```
 
-As we already know, JavaScript strings are based on [Unicode](https://en.wikipedia.org/wiki/Unicode): each character is represented by a byte sequence of 1-4 bytes.
+Como já sabemos, strings em JavaScript são baseadas em [Unicode](https://en.wikipedia.org/wiki/Unicode): cada caractere é representado por uma sequência de bytes de 1 a 4 bytes.
```

Comment:
It should be "já sabemos que, as Strings da JavaScript são baseadas no".

Reply:
I believe this change is also not necessary.

Since the word "linguagem" is omitted before "JavaScript", it can also be referred to with the preposition "em". 

This can also be based on the pt-br version of [MDN docs - Strings](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/String)

Reply:
The word **linguagem" is implied because JavaScript is a programming language.

And just because others do different does not make it right.